Doc/library/logging.handlers.rst

```diff
@@ -917,8 +917,7 @@ should, then :meth:`flush` is expected to do the flushing.
 
    .. method:: flush()
 
-      You can override this to implement custom flushing behavior. This version
-      just zaps the buffer to empty.
+      For a :class:`BufferHandler`, flushing means that it zaps the buffer to empty. This method can be overwritten to implement more useful flushing behavior.
```

Comment:
```suggestion
      For a :class:`BufferingHandler` instance, flushing means that it sets the buffer to an empty list. This method can be overwritten to implement more useful flushing behavior.
```

Reply:
True, thanks! I've also saw the wrong word for the class. I commit the suggestions.